bentoml/cli/bento.py

```diff
@@ -237,3 +245,112 @@ def retrieve(bento, target_dir):
         safe_retrieve(bento_service_bundle_path, target_dir)
 
         click.echo('Service %s artifact directory => %s' % (name, target_dir))
+
+    @cli.command(
+        help='Containerize given Bento into a Docker image',
+        short_help="Containerize given Bento into a Docker image",
+    )
+    @click.argument(
+        "bento", type=click.STRING, callback=parse_bento_tag_callback,
```

Comment:
Let's add the required flag since we need it to make docker image anyway

Reply:
If it's an argument, it should already be required right?
![image](https://user-images.githubusercontent.com/23178940/86411714-536a7200-bc72-11ea-95df-085bbb82183f.png)

Reply:
Oh it is an argument. Good catch!